nixos/modules/services/databases/postgresql.md

```diff
@@ -354,6 +354,27 @@ self: super: {
 }
 ```
 
+You can define a custom postgresql extension with `postgresqlBuildExtension` for C and sql extensions, or the `buildPgrxExtension` for pgrx (rust) extensions. It is then possible to use postgresql's `callPackage` method to build the extension:
```

Comment:
Let's stick to the "one sentence per line" rule mentioned in the docs' contribution guidelines and do some capitalization:


```suggestion
You can define a custom PostgreSQL extension with `postgresqlBuildExtension` for C and SQL extensions, or with `buildPgrxExtension` for pgrx (Rust) extensions.
It is then possible to use `postgresql`'s `callPackage` method to build the extension:
```

(somehow this still reads the wrong way around imho - you *first* use `callPackage`, and *then* you have access to the builder function, right?)